device-registry/src/main/scala/org/genivi/sota/device_registry/Routes.scala

```diff
@@ -122,48 +120,45 @@ class Routes(namespaceExtractor: Directive1[Namespace],
   def api: Route =
```

Comment:
wait what do you mean after devices? `system_info` is a resource of a device resource, so it makes sense that it's after a device id